posix.mak

```diff
@@ -279,7 +284,13 @@ install :
 	$(MAKE) -f $(MAKEFILE) OS=$(OS) MODEL=$(MODEL) BUILD=release INSTALL_DIR=$(INSTALL_DIR) \
 		DMD=$(DMD) install2
 
+ifeq (1,$(BUILD_WAS_SPECIFIED))
 unittest : $(addsuffix .d,$(addprefix unittest/,$(D_MODULES)))
+else
+unittest : unittest-debug unittest-release
+unittest-%: 
+	$(MAKE) -j4 unittest -f $(MAKEFILE) OS=$(OS) MODEL=$(MODEL) DMD=$(DMD) BUILD=$*
```

Comment:
That's what I meant.  I probably should have included the word 'invocation' after 'outer-most make'.  The auto-tester does indeed add -j<configured value> based on the number of cores the machine has (anywhere from 1 to 6 depending on the host).